packages/web/nexus/utils/stripe.ts

```diff
@@ -0,0 +1,7 @@
+import Stripe from 'stripe'
+
+const stripeConfig = new Stripe(process.env.STRIPE_SECRET_KEY || '', {
```

Comment:
not strictly relevant to here, but it just occurred to me that we should update the example .env file with dummy values and make sure you can still start up and run the code without stripe keys

Reply:
Ah yeah great point!